### ISSUE TITLE
No entities for dishwasher - API call failed with code=409 conflict

### ISSUE THREAD

**Issue by imlovingithub** [ekutner/home-connect-hass#324]
## Describe the bug
I'm not sure if this is a coincidence but the issue started following update to 1.0.2.  Following a failed automation run I noticed all the dishwasher entities were showing as "This entity is no longer being provided by the home_connect_alt integration. If the entity is no longer in use, delete it in settings."

I tried uninstalling and reinstalling home_connect_alt but it didn't help.  I have run debug on log mode 7 which is showing a 409 conflict error.

My HA is on OS 11.4 and core is on 2024.1.5

## Version of the custom_component
1.0.2

## Configuration
none

## Debug log
attached
[home-assistant_home_connect_alt_2024-01-21T14-27-48.564Z.log](https://github.com/ekutner/home-connect-hass/files/14001675/home-assistant_home_connect_alt_2024-01-21T14-27-48.564Z.log)

**Comment by ivanvaccari** [comment]
Same for me, but i've upgraded to 1.0.2 after the error, hoping it would solve it.

HA core 2023.11.0 on Docker

**Comment by ekutner** [comment]
This error indicates a connection problem between the appliance and the Home Connect cloud. These are the reasons HC list for it:
* HomeAppliance is offline
* HomeAppliance did not respond to connection initialization requests in time, it might be offline
* HomeAppliance did not respond in time, it might by offline

However, since it is happening to multiple people I suspect it is an issue with the HC cloud itself and they will fix it on their end. Either way, there is nothing I can do about it until it is fixed.

**Comment by imlovingithub** [comment]
Thanks @ekutner

I searched for home connect offline errors and found others had experienced similar issues in the past with the other Home Connect integration.  A restart of the device solved it for some of them and I can confirm that I have just restarted the dishwasher, reloaded home_connect_alt and it has now picked up all the entities again.

My apologies, I should have tried restarting the dishwasher before raising this issue but it was still working through the Home Connect app so I had no reason to believe it was offline.
